### PR TITLE
docs: explain why this gateway is being archived

### LINKED ISSUE #75: proposal: deprecate/archive this component
Impetus:

The [CloudEvents Gateway](https://github.com/brigadecore/brigade-cloudevents-gateway) will go GA very soon and works very well with [Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/overview). This means the ability to receive events from just about any Azure resource -- including Azure Container Registries -- already exists.

I'm not keen to incur the ongoing maintenance burden of a gateway that works with _one_ Azure service, when we already have a component that works with _most_ Azure services.

Can we?

This component isn't GA get. So, yes.

Flip side:

If you configure webhooks _directly_ on an ACR, you have the option to provide an image name pattern that will send webhooks only for image names that match the pattern. If you're using Event Grid to wire events from your ACR into Brigade, you don't have that option.

Is it worth maintaining a whole separate gateway for the sake of that one option? I would argue that the answer is, "No... unless demand for it surfaces."

Ultimately I'm proposing that we archive this repository, but remain open-minded to the possibility of resuming work on it in the future.

cc @vdice for opinion.

### PR DESCRIPTION
**Pull request by krancour**
Fixes #75.

This repository will be archived after this is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+> ⚠️&nbsp;&nbsp;Most Azure resources, including Azure Container Registries can
```
Comment (vdice) [inline README.md]:
```suggestion
> ⚠️&nbsp;&nbsp;Most Azure resources, including Azure Container Registries, can
```

Comment (vdice) [reply README.md]: Not sure if this is nit: style or hard grammar rule -- defer to you @krancour

**Review by vdice — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (krancour) [reply README.md]: I agree there should definitely be a comma there. Fixing now.

Comment (krancour) [reply README.md]: Fixed.

**Pull request merged**